pages/common/go-fmt.md

```diff
@@ -1,7 +1,7 @@
 # go fmt
 
 > Format Go source files.
```

Comment:
```suggestion
> Format Go source files and print the changed filenames.
```
Unfortunately it's not possible to apply suggestions on deleted lines, but how about this?